From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Employ an environment variable to access data.

To avoid confusion between built and original files, access the wiki
data that is not included amongst the built/installed files.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.
---
 src/moin/_tests/wikiconfig.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/moin/_tests/wikiconfig.py b/src/moin/_tests/wikiconfig.py
index 4998172..53adbb6 100644
--- a/src/moin/_tests/wikiconfig.py
+++ b/src/moin/_tests/wikiconfig.py
@@ -13,6 +13,7 @@ work without setting them (like data_dir).
 
 
 from os.path import abspath, dirname, join
+from os import environ
 
 from moin.config.default import DefaultConfig
 
@@ -21,7 +22,7 @@ class Config(DefaultConfig):
     """
     default configuration for the unit tests
     """
-    _here = abspath(dirname(__file__))
+    _here = abspath(environ["SRC_MOIN_TESTS"])
     _root = abspath(join(_here, '..', '..'))
     data_dir = join(_here, 'wiki', 'data')  # needed for plugins package TODO
     index_storage = 'FileStorage', (join(_here, 'wiki', 'index'), ), {}
